### ISSUE TITLE
Running on Windows 11 - onecore\net\netprofiles\service\src\nsp\dll\namespaceserviceprovider.cpp(550)\

### ISSUE THREAD

**Issue by vindberg** [Nethereum/Nethereum#973]
When running on windows 11. The GetHostByName is deprecated. Only GetHostName is supported as far as I can tell.

## Expected Behavior
No console warnings

## Current Behavior
Says: onecore\net\netprofiles\service\src\nsp\dll\namespaceserviceprovider.cpp(550)\nlansp_c.dll!00007FFE5424D93D: (caller: 00007FFE6AA6ACF6) LogHr(9) tid(634c) 8007277C No such service is known. The service cannot be found in the specified name space.

when doing a CallAsync()

## Possible Solution
Replace GetHostByName with GetHostName - https://stackoverflow.com/a/70978244

Using nugets 4.16 for dotnet core.

**Comment by juanfranblanco** [comment]
What is the context of your call, Nethereum does not use that method.

**Comment by vindberg** [comment]
Im simply getting the balance of an ERC20 token using the Web3 class of nethereum:

```
            var erc20Contract = web3.Eth.GetContract(tokenAbi.abi, token.Address);
            var balanceFunction = erc20Contract.GetFunction("balanceOf");
            return await balanceFunction.CallAsync<BigInteger>(address);

```
Running in debug mode im getting a thread exited with code 0 and the message outlined above.

**Comment by juanfranblanco** [comment]
Just a console application? You may be having an issue with vscode.

**Comment by vindberg** [comment]
Yeah its a hangfire job, but nevertheless it outputs this error when running in debug-mode. Im using Visual Studio.

It it's not your code causing it, then please just close this issue again.
